aldryn_forms/admin/base.py

```diff
@@ -30,10 +30,16 @@ def has_add_permission(self, request):
 
     def get_data_for_display(self, obj):
         data = obj.get_form_data()
-        html = render_to_string(
-            template_name='admin/aldryn_forms/display/submission_data.html',
-            dictionary={'data': data}
-        )
+        try:
+            html = render_to_string(
+                template_name='admin/aldryn_forms/display/submission_data.html',
+                dictionary={'data': data}
+            )
+        except TypeError:
+            html = render_to_string(
+                template_name='admin/aldryn_forms/display/submission_data.html',
+                context={'data': data}
+            )
```

Comment:
@stefanfoulis 
Agreed. Changed to https://github.com/aldryn/aldryn-forms/commit/3ee5ab4bca50af07554e330a8f74b286bca34ee7